LiveController_deployment.md

```diff
@@ -0,0 +1,91 @@
+## Machine prerequisites:
+====================================================
+liveController & liveRecorder can be installed in single or separate machines.
+
+## Software prerequisites:
+====================================================
+ - RHEL/CentOS 6.4 or above or Ubuntu 16.0.4 or above
+ - ffmpeg 3.0 -  (run build_scripts/build_ffmpeg.sh)
+
+
+ ## New installation of liveController (version format v[X.Y.Z]):
+ ===============================================================
+ # General
+ - it is recommended to install each version in directory created with the version name
+ and use symlink from latest to the currently used version
+
+ you can install liveController using deployment/upgradeLive or manually.
+
+ - install latest liveController version from git:
+ wget https://github.com/kaltura/liveDVR/archive/v[x.y.z].tar.gz
+ tar -xvzf v[x.y.z].tar.gz
+ mv liveDVR-[x.y.z] [installation base path]
+ rm -f v[x.y.z].tar.gz
+ - install Node Version Manager 6.3.0+
+ - install Node Packaged Modules (npm) 1.4.3+
+ - create configMapping.json from common/config/configMapping.json.template:
+
+    general configuration:
+    =======================================
+    @LIVE_CONTENT_PATH@ - root path to all live streams content
+ 	@LIVE_ARCHIVE_CONTENT_PATH@ - root path for archived live streams content
```

Comment:
Didn't w talk about using a general @ROOT_DIRECTORY@ path to decrease the amount of required tokens

Same applies to @RECORDING_FOLDER@, etc...